pinax/stripe/actions/accounts.py

```diff
@@ -1,9 +1,8 @@
-import stripe
-
+from . externalaccounts import sync_bank_account_from_stripe_data
 from .. import models
 from .. import utils
 import datetime
-from . externalaccounts import sync_bank_account_from_stripe_data
+import stripe
```

Comment:
Noticing some of the import ordering change around.  What's the reason for it?

In Pinax, we structure the imports like so:

```python
# first set of imports are stdlib imports
# non-from imports go first then from style import in their own group
import csv

# second set of imports are Django imports with contrib in their own
# group.
from django.urls import reverse
from django.db import models
from django.utils import timezone
from django.utils.translation import ugettext_lazy as _

from django.contrib.auth.models import User

# third set of imports are external apps (if applicable)
from tagging.fields import TagField

# fourth set of imports are local apps
from .fields import MarkupField
```

Details can be found in [CONTRIBUTING.md](https://github.com/pinax/pinax-stripe/blob/master/CONTRIBUTING.md).  I'm happy to fix them all up because I know at this point you just want to be done :).

Mostly curious as to why you wanted to make the change.

Reply:
Ah, sorry - I am in the habit of alphabetically ordering them and being done with it. I find this to be the lowest friction approach. But I should have stuck to your standard - thanks for patching that up.

Reply:
We (and Pinax in general) should have an isort config then (https://github.com/timothycrosley/isort).  There would be also flake8-isort to check them then, but the main benefit is that you can just `isort` yourself locally.